tests/useAnswersState.test.tsx

```diff
@@ -69,15 +101,15 @@ it('does not perform extra renders/listener registrations for nested components'
   expect(childStateUpdates).toHaveLength(0);
 
   userEvent.click(screen.getByText('Search'));
-  await pendingVerticalQuery;
+  await act( async () => { await pendingVerticalQuery; });
```

Comment:
out of curiosity why are these acts necessary?

Reply:
Oh ok, maybe this is due to changing from react 17 to 18 with how batching works or something. I think I wrote this original test in a pretty weird way but I couldn't think of an easy way to change it here so eh